pkg/validators/ocm_client.go

```diff
@@ -0,0 +1,39 @@
+package validators
+
+import (
+	"context"
+
+	amv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+// OCMError abstracts behavior required for validators to identify the underlying
+// causes of OCM related errors.
+type OCMError interface {
```

Comment:
Will we have various implementation of `OCMError`? Seems to me we need a type + method receiver.